docs/schema/animation/animation.json

```diff
@@ -76,9 +80,24 @@
                     "title": "Metadata",
                     "description": "Document metadata",
                     "$ref": "#/$defs/animation/metadata"
+                },
+                "markers": {
+                    "title": "Markers",
+                    "description": "Markers defining named sections of the composition.",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/$defs/helpers/marker"
+                    }
+                },
+                "mb": {
+                    "title": "Motion Blur",
+                    "$ref": "#/$defs/animation/motion-blur"
                 }
             },
             "required": ["w", "h", "fr", "layers", "op", "ip"]
+        },
+        {
+            "$ref": "#/$defs/animation/composition"
```

Comment:
I'm a bit confused by the diff here, it shows some changes that are already there. The merge request shows conflicts too so maybe it's based on an older commit?

Could you merge with the latest main?

Reply:
Yeah, hold on
I was just trying to add makers.